pystiche_papers/gatys_ecker_bethge_2015/_loss.py

```diff
@@ -25,7 +25,9 @@ def __init__(
     ):
         super().__init__(encoder, score_weight=score_weight)
 
+        # https://github.com/pmeier/PytorchNeuralStyleTransfer/blob/master/NeuralStyleTransfer.ipynb [3]
         self.score_correction_factor = 1.0 if impl_params else 1.0 / 2.0
+        # https://github.com/pmeier/PytorchNeuralStyleTransfer/blob/master/NeuralStyleTransfer.ipynb [9]
```

Comment:
I don't see what cell `9` has to do with it. This should also reference `torch.nn.MSELoss()` which by default uses `reduction="mean"`.